Mathlib/Logic/Function/Basic.lean

```diff
@@ -696,15 +696,20 @@ attribute [local instance] Classical.propDecidable
 
 variable {α β γ : Sort*} {f : α → β}
 
-/-- `extend f g e'` extends a function `g : α → γ`
-along a function `f : α → β` to a function `β → γ`,
-by using the values of `g` on the range of `f`
-and the values of an auxiliary function `e' : β → γ` elsewhere.
```

Comment:
Nit: I quite liked the mnemonic of `e` for `elsewhere`

Reply:
Oh I really didn't get that mnemonic. I used `j` for junk.

Reply:
Sorry, I don't really see how to restore that mnemonic.